demos/dermatology.html

```diff
@@ -193,9 +193,14 @@
         var currWidget = null;
         var showProperties = null;
         var debouncedResize = null;
+        var propEditor = null;
 
         require(["src/layout/Surface", "src/layout/Grid", "src/other/Persist", "src/other/PropertyEditor"], function (Surface, Grid, Persist, PropertyEditor) {
-            var propEditor = new PropertyEditor().showColumns(false).showData(false);
+            propEditor = new PropertyEditor()
+                    .showColumns(false)
+                    .showData(false)
+                    .excludeTags(["Advanced","Private"])

```

Comment:
Then should it default to **[]** in dermatology?

Currently we could change it on the dermatology page by executing **propEditor.excludeTags(["Advanced","Private"])** in the console.